R/dbConnect__duckdb_driver.R

```diff
@@ -55,7 +58,8 @@ dbConnect__duckdb_driver <- function(
   timezone_out = "UTC",
   tz_out_convert = c("with", "force"),
   config = list(),
-  bigint = "numeric"
+  bigint = "numeric",
+  array = "none"
```

Comment:
I don't think `bigint` belongs here, this should be deprecated at some point. Let's not add `"array"` here. Should have been clearer about that in the code, add a comment to that effect now?

Reply:
I'm mistaken, this is where `array = "none"` belongs. For some reason I thought this is the `duckdb()` function.